<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FGEAnimationSelectionForm</class>
 <widget class="QWidget" name="FGEAnimationSelectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid ;
	border-color: rgb(156, 156, 156);
    margin-top: 10px;
    font-size: 14px;
    border-radius: 3px;
}

QGroupBox::title {
	top:0px;
	left : 15px;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px 2px;
}

QPushButton {
	background-color: rgb(176, 80, 255);
	border-style: outset;
	border-width: 1px;
	border-radius: 2px;
	border-color: rgb(17, 17, 17);
}

QPushButton:pressed {
      background-color: rgb(224, 0, 0);
      border-style: inset;
}


QMenu {
	  color: rgb(0, 0, 0);
      border-radius: 3px;
      background-color: white;
      margin: 2px; /* some spacing around the menu */
  }

  QMenu::item {
	height : 30px;
      padding: 2px 25px 2px 20px;
      border: 1px solid transparent; /* reserve space for selection border */
  }

  QMenu::item:selected {
		border-radius: 3px;
		background-color: rgb(235, 235, 235);
  }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="3" column="2">
    <widget class="QPushButton" name="SelectPushButton">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>12</number>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Animations List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="AddNewAnimPushButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/Icons/svg/up_white.svg</normaloff>:/Icons/svg/up_white.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>205</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="FGEListWidget" name="AnimListWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::AllEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Channels List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="FGEListWidget" name="ChannelsListWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>101</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>104</height>
      </size>
     </property>
     <property name="title">
      <string>Descreption</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>channel count :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="ChannelCountLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>error :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="ErrorLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>127</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="ClosePushButton">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FGEListWidget</class>
   <extends>QListWidget</extends>
   <header>GUI/ListWidget/FGEListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
